packages/fuselage-ui-kit/src/elements/RadioButtonElement.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, RadioButton } from '@rocket.chat/fuselage';
+import * as UiKit from '@rocket.chat/ui-kit';
+import { type ReactElement } from 'react';
+
+import { useUiKitState } from '../hooks/useUiKitState';
+import type { BlockProps } from '../utils/BlockProps';
+
+type RadioButtonElementProps = BlockProps<UiKit.RadioButtonElement>;
+
+const RadioButtonElement = ({
+  block,
+  context,
+  surfaceRenderer,
+}: RadioButtonElementProps): ReactElement => {
+  const [{ loading }, action] = useUiKitState(block, context);
```

Comment:
Here we have to implement the control of the value in a specific way: radio select must be a single value selector. Each selection must remove another previous one.